src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSliderUI.java

```diff
@@ -151,6 +151,7 @@ public class BasicSliderUI extends SliderUI{
     /**
      * Constructs a {@code BasicSliderUI}.
      */
+    @Deprecated(since = "23")
```

Comment:
You need to add `forRemoval = true` to deprecate this for removal. Also, we typically add a `@deprecated` javadoc tag in addition to the `@Deprecated` annotation.

Reply:
Added forRemoval and @deprecated